docs/UserGuide.md

```diff
@@ -113,40 +113,40 @@ display box, which is located immediately below the command box.
 
 The main viewing area consists of two sections: **Classes** and **Students**.
 
-* The **Classes** section contains information about classes and lessons in Tutor's Pet. To view the list of all
- classes in Tutor's Pet, make use of the **`list-class`** command.
+* The **Classes** section contains information about classes and lessons in Tutor's Pet. Use the **`list-class`** command to view the list of all
+classes in Tutor's Pet.
 
     ![Class_section](images/ugimages/ClassPanel.png)
     <br>Figure 4. The class section.
 
-* The **Students** section displays contains information about students in Tutor's Pet. To view the list of all
- students in Tutor's Pet, make use of the **`list-student`** command.
+* The **Students** section displays contains information about students in Tutor's Pet. Use the **`list-student`** command to view the list of all students in Tutor's Pet.
 
     ![Student_section](images/ugimages/StudentPanel.png)
     <br>Figure 5. The student section.
 
 The utility area consists of three tabs: **File**, **Theme**, **Help**.
 
-* The **File** tab consists of an exit button. To exit Tutor's Pet, click on the exit button. Alternatively, make
- use of the `exit` command to exit the application.
+  ![Utility area](images/ugimages/UtilityArea.png)
+  <br>Figure 6. The utility area.
 
-* The **Theme** tab consist of 3 different themes, mainly **Light**, **Alternate** and **Dark**. To change the theme of
+* The **File** tab consists of an exit button. To exit Tutor's Pet, click on the exit button. Alternatively, use the `exit` command to exit the application.
+
+* The **Theme** tab consist of 3 different themes: **Light**, **Alternate** and **Dark**. To change the theme of
```

Comment:
Should we show her the different themes as well? E.g using screenshots

Reply:
Screenshot just the dropdown menu of the theme button?

Reply:
Was thinking of the different themes (ie light, alternate, dark) and not just the buttons alone. What do you think?